acceptance_tests/features/steps/refusal.py

```diff
@@ -30,7 +30,7 @@ def send_bad_refusal_message(context):
     context.sent_messages.append(message)
 
 
-def _send_refusal_message(correlation_id, originating_user, case_id):
+def _send_refusal_message(correlation_id, originating_user, case_id, erase_data):
```

Comment:
Not all callers of this have been updated with this signature change, breaks other scenarios

Reply:
Defaulted to false now so should be fixed 👍